documentation/src/docs/pravega-concepts.md

```diff
@@ -19,7 +19,8 @@ Pravega concepts.
 
 Pravega organizes data into Streams.  A Stream is a durable, elastic, append-only, unbounded sequence of bytes that has good performance and strong consistency.  A Pravega Stream is
 similar to but more flexible than a "topic" in popular message-oriented middleware such as
-[RabbitMQ](https://www.rabbitmq.com/) or Apache Kafka.
+[RabbitMQ](https://www.rabbitmq.com/) or [Apache Kafka](https://kafka.apache.org/) in a way that MOM systems mainly focus on message "delivery", whereas Pravega targets how these 
+messages (or events) are consistently and efficiently "stored" for short term (low latency) and long term (high throughput) consumption.
```

Comment:
I like adding here few words that differentiate Pravega from other systems. My only concern is that Kafka is not just a message broker (as RabbitMQ), given that it also provides storage: 
>Store streams of records in a fault-tolerant durable way.

In my humble opinion, the current sentence fits the comparison with MOMs, but perhaps we would need to add a separate line differentiating Pravega from Kafka.